index.html

```diff
@@ -81,146 +81,263 @@
         maxTocLevel: 4,
         inlineCSS: true
       };
-
-      var includeTerms = [ "identity profile", "entity", "entity credential", "subject", "claim" ];
     </script>
   </head>
   <body>
     <section id='abstract'>
       <p>
-A self-sovereign architecture for verifiable claims is one where the
-holder of a verifiable claim is in complete control of their
-identifier, where their verifiable claims are stored, and how they are
-used. There is currently no widely used self-sovereign,
-privacy-enhancing standard for expressing and transacting verifiable
-claims (aka: credentials, attestations) via the Web.
+We typically use claims made by others about us to gain access to
+various online services. For example, we use
+email addresses to identify ourselves to online services,
+driver's licenses to prove that we are capable of operating a
+motor vehicle, university degrees to prove we are well-trained
+and knowledgeable, and government-issued passports to travel
+between countries and access financial services. It is the
```

Comment:
Opening the document with a promise to "gain access to various online services" seems like the antithesis of what our goal and charter is.  That smacks of "credentials = authentication" and it makes me fairly uncomfortable.  It doesn't set the right context for what we're actually doing.

The rest of the paragraph is better -- proving we can drive a car, being well trained and knowledgable, travel between countries, and access to financial services are great.

How about striking "We typically use claims made by others about us to gain access to various online services" and replacing it with:

Regardless of the digital or physical space, nearly every interaction where one individual is asking another for permission, access, or a benefit relies on trust and the ability of the individual to make a claim that the other can verify.